workers/data_refinery_workers/processors/create_quantpendia.py

```diff
@@ -19,25 +20,19 @@
 SMASHING_DIR = "/home/user/data_store/smashed/"
 
 logger = get_and_configure_logger(__name__)
-
+logger.setLevel(logging.getLevelName('DEBUG'))
 
 def create_quantpendia(job_id: int) -> None:
     pipeline = Pipeline(name=PipelineEnum.CREATE_QUANTPENDIA.value)
     job_context = utils.run_pipeline({"job_id": job_id, "pipeline": pipeline},
                                      [utils.start_job,
                                       download_files,
                                       create_result_objects,
+                                      remove_job_dir,
                                       utils.end_job])
     return job_context
 
 
-def get_process_stats():
-    BYTES_IN_GB = 1024 * 1024 * 1024
-    process = psutil.Process(os.getpid())
-    ram_in_GB = process.memory_info().rss / BYTES_IN_GB
-    return { 'total_cpu': psutil.cpu_percent(), 'process_ram': ram_in_GB }
-
-
 def download_files(job_context: Dict) -> Dict:
     job_context['time_start'] = timezone.now()
     _make_dirs(job_context)
```

Comment:
```suggestion
    job_context = _make_dirs(job_context)
```
You set some properties in there. Sure this is modifying `job_context` so it's not truly necessary, but doing this will make it explicit that `job_context` is being updated by `_make_dirs`.

Reply:
good point! moved `make_dirs` into one of the functions in `run_pipeline`